src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE `movie_app`

```

Comment:
where are other queries?

Reply:
returne quieries and fix " AUTO_INCREMENT=1 "